examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,225 @@
+#
+# .. figure:: ../ensemble/images/sphx_glr_plot_gradient_boosting_categorical_001.png
+#   :target: ../ensemble/plot_gradient_boosting_categorical.html
+#   :align: center
+
+##############################################################################
+# Improved performances in HistGradientBoosting methods
+# -----------------------------------------------------
+# Histogram initialization is now done in parallel in
+# :class:`ensemble.HistGradientBoostingRegressor` and
+# :class:`ensemble.HistGradientBoostingClassifier` which results in speed
+# improvement.
+# See more in the `Benchmark page
+# <https://scikit-learn.org/scikit-learn-benchmarks/>`_.
+
+##############################################################################
+# New SequentialFeatureSelector transformer
+# -----------------------------------------
+# A new iterative transformer to select features is available:
+# :class:`~sklearn.feature_selection.SequentialFeatureSelector`.
+# Sequential Feature Selection can add features once at a time (forward
+# selection) or remove features from the list of the available features
+# (backward selection), based on a cross-validated score maximization.
+# See the :ref:`User Guide <sequential_feature_selection>`.
+
+from sklearn.feature_selection import SequentialFeatureSelector
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.datasets import load_iris
+
+X, y = load_iris(return_X_y=True, as_frame=True)
+feature_names = X.columns
+knn = KNeighborsClassifier(n_neighbors=3)
+sfs = SequentialFeatureSelector(knn, n_features_to_select=2)
+sfs.fit(X, y)
+print("Features selected by forward sequential selection: "
+      f"{feature_names[sfs.get_support()]}")
+
+##############################################################################
+# New PolynomialCountSketch kernel approximation function
+# -------------------------------------------------------
+# The new :class:`~sklearn.kernel_approximation.PolynomialCountSketch`
+# approximates a polynomial expansion of a feature space when used with linear
+# models, but uses much less memory than
+# :class:`~sklearn.preprocessing.PolynomialFeatures`, and is often faster than
+# using a polynomial kernel in SVM.
+
+from sklearn.datasets import fetch_covtype
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import train_test_split
+from sklearn.kernel_approximation import PolynomialCountSketch
+from sklearn.svm import LinearSVC
+
+X, y = fetch_covtype(return_X_y=True)
+pipe = make_pipeline(PolynomialCountSketch(degree=4, n_components=1000),
+                     LinearSVC())
+X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=5000,
+                                                    test_size=10000,
+                                                    random_state=42)
+pipe.fit(X_train, y_train)
+
+##############################################################################
+# Retrieving datasets from literature as pandas dataframes
```

Comment:
Not worth a highlight IMHO. From what I read on the whatsnew, we only added support for a few more `fetch_` utilities?

Reply:
Same here. I would also exlucde from highlighting.